bin/rucio

```diff
@@ -293,8 +316,23 @@ def whoami_account(args):
     """
     client = get_client(args)
     info = client.whoami()
-    for k in info:
-        print(k.ljust(10) + ' : ' + str(info[k]))
+    if cli_config == 'rich':
+        keyword_styles = {**CLITheme.ACCOUNT_STATUS, **CLITheme.ACCOUNT_TYPE} if console.is_terminal else {}
+        table_data = [(k, Text(str(v), style=keyword_styles.get(str(v), 'default'))) for (k, v) in sorted(info.items())]
+        table = generate_table(table_data, col_alignments=['left', 'left'], row_styles=['none'])
+        if console.is_terminal:
+            if args.no_pager:
+                console.print(table)
+            else:
+                console.width = sys.maxsize
+                console.begin_capture()
+                console.print(table)
+        else:
+            console.width = sys.maxsize
+            console.print(table)
```

Comment:
Isn’t this sequence of statements repeated multiple times? It looks to me like it should become a helper function.